packages/client/src/v2-events/components/forms/utils.ts

```diff
@@ -147,38 +144,53 @@ export function fieldValueToString({
   value: FieldValue
   intl: IntlShape
   locations: { [locationId: string]: ILocation }
-}) {
+}): string {
   switch (fieldConfig.type) {
+    case FieldType.BULLET_LIST:
+    case FieldType.DIVIDER:
+    case FieldType.PAGE_HEADER:
     case FieldType.DATE:
-      return dateToString(value as DateFieldValue)
     case FieldType.TEXT:
-      return textToString(value as TextFieldValue)
     case FieldType.PARAGRAPH:
-      return paragraphToString(value as ParagraphFieldValue)
+    case FieldType.FILE:
+      // @TODO:
+      return defaultTo(value as string, '')
+
     case FieldType.CHECKBOX:
-      return checkboxToString(value as CheckboxFieldValue)
+      // @TODO: This should be a boolean?
+      return value === 'true' ? 'Yes' : 'No'
     case FieldType.RADIO_GROUP:
-      return selectFieldToString(
-        value as RadioGroupFieldValue,
-        fieldConfig.optionValues,
-        intl
+    case FieldType.SELECT: {
+      const selectedOption = fieldConfig.options.find(
+        (option) => option.value === value
       )
-    case FieldType.SELECT:
-      return selectFieldToString(
-        value as SelectFieldValue,
-        fieldConfig.options,
-        intl
+
+      return selectedOption ? intl.formatMessage(selectedOption.label) : ''
```

Comment:
Is this correct? I'm guessing we do not want to store formatted values in the form data instead, right?